packages/common/src/ui/components/NavLink.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
```

Comment:
Just copy pasted.

I would like to try and use the styled-components style for this 😁